private/protocol/jsonrpc/unmarshal_test.go

```diff
@@ -1166,6 +1166,26 @@ func TestOutputService3ProtocolTestTimestampMembersCase1(t *testing.T) {
 
 }
 
+func TestOutputService3ProtocolTestTimestampMembersCase2(t *testing.T) {
```

Comment:
Ah, I missed that — I had seen the note in CONTRIBUTING.md about `model` but didn't realize this file was generated from there, though it makes sense in retrospect. Maybe the CONTRIBUTING.md note could mention the other files that are generated?